dev/com.ibm.ws.microprofile.opentracing.jaeger.adapter/resources/com.ibm.ws.microprofile.opentracing.jaeger.resources/JaegerMessages.nlsprops

```diff
@@ -0,0 +1,84 @@
+# 2) A equals ("=") sign separating the key and text.  This is required.
+# 3) A string containing data that will be displayed to
+#    the user when running the application.  This is required and
+#    SHOULD be translated.
+#
+# In short, each line in the resource array contains two strings.
+# The SECOND quoted string on each line should be translated.
+#
+# Some of the strings to be translated contain special codes
+# for data that will be inserted into the string when the string
+# is displayed to the user.  For example, say you have the
+# following string to translate:
+#
+#   This file was written by the IBM Administrator on {0,date}.
+#
+# {0,date} is a code that tells java to insert a parameter into
+# this space when displaying it to the user.  In this case,
+# the parameter that is inserted is a date.
+#
+# The 0 indicates that this is where the first parameter is
+# inserted in the string.  There may be more than one parameter
+# in any one string.  The 'date' tells java to format this
+# parameter as a date.
+#
+# The {0,date} code should be placed at the appropriate position
+# in the translated text.  Please note that the code in the
+# translation must match exactly the code in the original
+# string.
+#
+# The developer should describe in comments above the string to be
+# translated each of the parameters so that you will be able to
+# determine where the best location for each parameter in the
+# translated string.
+#
+# Lines that begin with a pound sign (#) are comments and should not be
+# translated.
+# -----------------------------------------------------------------------
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by Opentracing Impl -  range is 0000-4999, plus 9999 for the emergency message
+# This bundle uses range 1000-1100
+#-----------------------------------------------------------------------------------------------------------------------------
+JAEGER_TRACER_CREATED=CWMOT1001I: An instance of JaegerTracer has been created for application {0}.  Tracing information is sent to {1}.
+JAEGER_TRACER_CREATED.explanation=An instance of JaegerTracer has been created to send spans from the application to the tracing backend.
+JAEGER_TRACER_CREATED.useraction=No action required.
+
+JAEGER_ENV_VAR_PARSE_ERROR=CWMOT1002W: The value of system property or environment variable {0} is not a valid type of {1}.
+JAEGER_ENV_VAR_PARSE_ERROR.explanation=The value defined in the system property or environment variable cannot be converted to the type.
+JAEGER_ENV_VAR_PARSE_ERROR.useraction=Define a value for the system property or environment variable that matches the type. 
+
+JAEGER_PASSWORD_CANNOT_DECODE=CWMOT1003W: The value of JAEGER_PASSWORD cannot be decoded.
```

Comment:
Change
"The value of JAEGER_PASSWORD cannot be decoded."
To
"The value of the JAEGER_PASSWORD environment variable cannot be decoded."